static/app/components/onboarding/gettingStartedDoc/utils/useLoadGettingStarted.tsx

```diff
@@ -0,0 +1,57 @@
+import {useEffect, useState} from 'react';
+import * as Sentry from '@sentry/react';
+
+import type {Docs} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {
+  feedbackOnboardingPlatforms,
+  replayPlatforms,
+} from 'sentry/data/platformCategories';
+import type {PlatformKey} from 'sentry/types/project';
+
+export default function useLoadGettingStarted({
+  platformId,
+  platformPath,
+  replayPlatformCheck,
+  feedbackPlatformCheck,
+}: {
+  platformId: PlatformKey;
+  platformPath: string;
+  feedbackPlatformCheck?: boolean;
+  replayPlatformCheck?: boolean;
+}) {
+  const [module, setModule] = useState<
+    | null
+    | {
+        default: Docs<any>;
+      }
+    | 'none'
+  >(null);
+
+  useEffect(() => {
+    async function getGettingStartedDoc() {
+      if (replayPlatformCheck && !replayPlatforms.includes(platformId)) {
+        setModule('none');
+        return;
+      }
+      if (feedbackPlatformCheck && !feedbackOnboardingPlatforms.includes(platformId)) {
+        setModule('none');
+        return;
+      }
```

Comment:
```suggestion
      if ((productType === 'replay' && !replayPlatforms.includes(platformId)) || (productType === 'feedback' && !feedbackOnboardingPlatforms.includes(platformId))) {
        setModule('none');
        return;
      }
```
something like this could remove a little duplication (sorry for bad formatting)